karaf/Dockerfile.rhel

```diff
@@ -27,7 +27,6 @@ USER root
 ADD jboss.repo /etc/yum.repos.d/jboss.repo
 RUN yum install -y --disablerepo=\* --enablerepo=jboss maven \
  && rpm -e --nodeps java-1.7.0-openjdk-devel java-1.7.0-openjdk java-1.7.0-openjdk-headless \
```

Comment:
If `yum remove` can remove forcefully, then this could be an option, too. Need to remove this because Maven 3.0.5 depends hard on Java 7 but we need Java 8. We could switch the alternatives, but removing Java 7 also makes the image much smaller is imo a bit saver, too.

What is the advantage of `yum remove` ?